docs/docs/guides/web3_migration_guide.md

```diff
@@ -63,3 +63,21 @@ batch.execute();
 // in 4.x
 const responses = await batch.execute();
 ```
+
+### Web3-eth-accounts
+
+#### create and wallet.create
+
+In 1.x `entropy` is an optional parameter for the create method, 4.x does not have `entropy` as a parameter, create method uses an audited package ethereum-cryptography/secp256k1 that is cryptographically secure random number with certain characteristics.
+
+#### privateKeyToAccount
+
+In 1.x `ignoreLength` is an optional parameter for privateKeyToAccount, in 4.x `ignoreLength` is removed
```

Comment:
Don't you think we should fix this to avoid compatibility issues with 1.x?

Reply:
initially we removed this because it was error prone, but I think ur right for compatibility